### DISCUSSION TITLE
Scrolling (Mpris/media) Text

### DISCUSSION THREAD

**Discussion by Rido-o** [Aylur/ags#572] [Q&A]
Is it possible to create a bar widget that has the currently playing music text continuously scroll if the text length is longer than it's box.

**Comment by rubiin** [comment]
+1 on this

**Comment by enessmr** [comment]
maybe, but i did a similar one where the box EXPANDS instead of the scrolling text

**Comment by kotontrion** [comment]
See https://github.com/Aylur/astal/pull/291
